azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -120,3 +120,20 @@ CREATE UNIQUE INDEX version_set_md5
 -- alter table execution_flows add column dispatch_method TINYINT default 1;
 -- CREATE INDEX ex_flows_dispatch_method ON execution_flows (dispatch_method);
 
+-- Definition for hp_flow_owners table. It contains list of owners who have
```

Comment:
Is there a design doc for the db tables, dao and service?

Reply:
I didn't see a method to update the hp_flow_owners table, such as insert or delete. Also it might be useful if we add a state (active 1 vs inactive 0) col for the owners table.

Reply:
As earlier stated, the original intent of this PR was to get basic functionality out and this was supposed to be followed by implementation for ownership mapping. However, this did not go in time due to other factors and the rest of the implementation will be done in this PR only.